components/tooltip/template.html

```diff
@@ -0,0 +1,5 @@
+<!--Copyright (c) Coherent Labs AD. All rights reserved. Licensed under the MIT License. See License.txt in the project root for license information. -->
+<div class="tooltip">
+    <!-- TODO:scrollbar -->

```

Comment:
Remove (check the general comment for a task idea).

Reply:
Removed.